go/kbfs/libkbfs/leveldb.go

```diff
@@ -35,19 +35,22 @@ var leveldbOptions = &opt.Options{
 	OpenFilesCacheCapacity: 10,
 }
 
-type levelDb struct {
+// LevelDb is a libkbfs wrapper for leveldb.DB.
+type LevelDb struct {
```

Comment:
If we're going to export it, then it might as well go into its own package.

Reply:
The linter yelled at me for using it in an exported function and not exporting it. It may be possible to un-export that function instead though.

Reply:
`GetConflictResolutionDB` which is in an interface right now so probably can't be simply unexported. But maybe there's a better way to have a single CR DB that exists always and all CRs can access than putting it on the config.

Reply:
OK feel free to leave it then.

Reply:
@jzila do you have package name opinions? `kbldb`?

Reply:
`kbfsleveldb` probably, although its patterns could be universally useful.